src/ConnectorPermissions.ts

```diff
@@ -0,0 +1,83 @@
+import * as nearAPI from "near-api-js";
+import {KeyPair} from "near-api-js";
+import { Contract } from "near-api-js";
+import {Environment, Network, environmentToContractNameInfix, getNetworkId} from "./Utils";
+const { keyStores } = nearAPI;
+const { connect } = nearAPI;
+import { PERMISSIONS_CONTRACT_PREFIX, CALIMERO_CONTRACT_SUFFIX } from "./Constants";
+import { getNearConfig } from "./nearConfig";
+
+export class ConnectorPermissions {
+
+  shardName: string;
+  permissionsContract: Contract;
+
+  private constructor(shardName: string, permissionsContract: nearAPI.Contract)
+  {
+    this.shardName = shardName;
+    this.permissionsContract = permissionsContract;
+  }
+  
+  static async init(shardName: string,
+    env: Environment,
+    network: Network,
+    accountId: string,
+    keyPair: KeyPair): Promise<ConnectorPermissions> {
```

Comment:
I would also have an option that any method can be called without providing the private key - by constructing the transaction and leaving it to an external service to sign the tx (e.g. in browser wallet), but this PR is fine for now without it
Such methods would return the raw transaction, the user would need to sign it on their own and then the signedTx could be sent to the near network